src/main/java/com/hms_networks/americas/sc/extensions/eventfile/EventFile.java

```diff
@@ -16,6 +16,12 @@
  * @since 1.6.0
  */
 public class EventFile {
+  /** Event File line delimiter */
```

Comment:
File does not need to be capitalized. This is repeated throughout the code on this and the next commit

Reply:
Agreed, although relatively minor